internal/app/orchestrator/orchestrator.go

```diff
@@ -122,6 +126,7 @@ func (o *orchestrator) CreateWatch(req gcp.Request) (chan gcp.Response, func())
 	// If this is the first time we're seeing the request from the
 	// downstream client, initialize a channel to feed future responses.
 	responseChannel := o.downstreamResponseMap.createChannel(&req)
+	o.scope.Counter("create_channel").Inc(1)
```

Comment:
Can you make this stats name a const? Ideally we'd also subscope this to `"downstream"` and have the downstreamResponseMap increment this in the `createChannel` function.

Reply:
good idea.

I added this here just to test, but it makes sense to add it for real, using the convention we're going to use going forward.